### PR TITLE
Add bnl homebrew formula

### PR DESCRIPTION
**Pull request by jewilmeer**
First steps in a blendle wide CLI platform 🎉

### CODE REVIEW

**Review by soudy — APPROVED** [review]

**Review by eterps — APPROVED** [review]

File: Formula/bnl.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Bnl < Formula
+  desc "Blendle Command Line Tools"
+  homepage "https://github.com/blendle/bnl"
+  url "https://github.com/blendle/bnl.git",
+    version: "v.0.1.0",
+    revision: "a77f7ad417ff8a82d03c666e4ae1dc424b3a7b1e"
+  version "a77f7ad417ff8a82d03c666e4ae1dc424b3a7b1e"
+  head "https://github.com/blendle/bnl.git"
+
+  depends_on "fzf"
```
Comment (koenbollen) [inline Formula/bnl.rb:10]: Really?

Comment (jewilmeer) [reply Formula/bnl.rb:10]: ~~auto complete...~~

Comment (eterps) [reply Formula/bnl.rb:10]: `bnl` could also check if user has fzf installed and use it if that's the case (if you use fzf bij default it is also harder to script/automate)

Comment (koenbollen) [reply Formula/bnl.rb:10]: nvm, is just didn't expected it here and though it was a copy/paste mishap

**Pull request merged**